src/core/Akka.Remote.Tests/RemoteConsistentHashingRouterSpec.cs

```diff
@@ -45,6 +45,36 @@ public void ConsistentHashingGroup_must_use_same_hash_ring_independent_of_self_a
             var result2 = keys.Select(k => consistentHash2.NodeFor(k).Routee);
             Assert.Equal(result2,result1);
         }
+
+        [Fact]
+        public void ConsistentHashingGroup_must_use_same_hash_ring_independent_of_local_and_remote_nodes()
+        {
+            var a1 = new Address("akka.tcp", "RemoteConsistentHashingRouterSpec-1", "client1", 2552);
+            var a2 = new Address("akka.tcp", "RemoteConsistentHashingRouterSpec-1", "client2", 2552);
+            var localActor = Sys.ActorOf(Props.Empty, "a");
+
+            var s1 = new ActorRefRoutee(localActor);
+            var s2 = new ActorSelectionRoutee(Sys.ActorSelection("akka.tcp://RemoteConsistentHashingRouterSpec-1@client2:2552/user/a"));
+            var nodes1 = new List<ConsistentRoutee>(new[] { new ConsistentRoutee(s1, a1), new ConsistentRoutee(s2, a1) });
+
+            var s4 = new ActorSelectionRoutee(Sys.ActorSelection("akka.tcp://RemoteConsistentHashingRouterSpec-1@client1:2552/user/a"));
+            var s5 = new ActorRefRoutee(localActor);
+
+            var nodes2 = new List<ConsistentRoutee>(new[] {new ConsistentRoutee(s5, a2), new ConsistentRoutee(s4, a2)});
+
+            var consistentHash1 = ConsistentHash.Create(nodes1, 10);
+            var consistentHash2 = ConsistentHash.Create(nodes2, 10);
+            var keys = new List<string>(new[] { "A", "B", "C", "D", "E", "F", "G" });
+
+            var result1 = keys
+                .Select(k => consistentHash1.NodeFor(k))
+                .Select(routee => routee.ToString());
+
+            var result2 = keys
+                .Select(k => consistentHash2.NodeFor(k))
+                .Select(routee => routee.ToString());
+
+            Assert.Equal(result2, result1);
```

Comment:
This does a `.SequenceEqual` call under the covers?

Reply:
I think so. I can do a FluentAssertion here that's much clearer.